lib/cpg.c

```diff
@@ -986,6 +986,7 @@ cs_error_t cpg_zcb_alloc (
 		sizeof (struct qb_ipc_response_header));
 
 	if (error != CS_OK) {
+		munmap (buf, map_size);
```

Comment:
Do we also need to unlink(path) here? see above error handler in this file.

Reply:
note to myself - github "resolve conversation" doesn't store comment :(. So again. The thing is, where the failure happens. If failure happens before/during IPC request and before IPC reply, file should be deleted. But if failure happens during IPC reply then corosync already deleted the file so library could delete non-existing file (not a problem) or (very theoretical situation) delete file created by some other application. That's why I've decided to rather be on safe side. No matter what, I think this needs some improvement so I will merge rest of the patches and send better PR for last one.